tools/bktool/main.c

```diff
@@ -882,7 +882,7 @@ int main(int argc, char *argv[]) {
         } else if(keylist->count > 0) {
             bk_keylist();
         } else {
-            bk_info(&bk);
+            bk_getinfo(&bk);
```

Comment:
This change seems to be maybe part of some other commit?

Reply:
it was renamed to not conflict with the struct of the same name.

Reply:
I don't object to the change, but it doesn't seem to be in the correct commit.

Reply:
Moved it into its own commit :+1: